src/bin/ion/commands/generate/templates/java/sequence.templ

```diff
@@ -28,6 +28,26 @@ class {{ model.name }} {
         return;
     }
 
+     public static class {{ model.name }}Builder {
```

Comment:
Sequences don't need a builder. Just have the user pass in a `List` to the constructor. You only need a builder when there are named fields and there's more than one field.